dids/src/test/kotlin/web5/sdk/dids/methods/jwk/DidJwkTest.kt

```diff
@@ -12,19 +10,22 @@ import web5.sdk.common.Convert
 import web5.sdk.common.Json
 import web5.sdk.crypto.AlgorithmId
 import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.crypto.JwaCurve
+import web5.sdk.crypto.jwk.Jwk
 import web5.sdk.dids.DidResolutionResult
 import web5.sdk.dids.DidResolvers
+import web5.sdk.dids.exceptions.InvalidMethodNameException
+import web5.sdk.dids.methods.dht.DidDht
 import web5.sdk.testing.TestVectors
 import java.io.File
 import kotlin.test.assertEquals
 import kotlin.test.assertNotNull
-import kotlin.test.assertTrue
 
 class DidJwkTest {
   @Nested
   inner class CreateTest {
     @Test
-    fun `creates an ES256K key when no options are passed`() {
+    fun `creates an EdDSA key when no options are passed`() {
```

Comment:
```suggestion
    fun `creates an Ed25519 key when no options are passed`() {
```

EdDSA is an algorithm not a key 😄